etc/WMAgentConfig.py

```diff
@@ -141,6 +141,7 @@
 # this has to be unique for different work queue. This is just place holder
 config.WorkQueueManager.queueParams["QueueURL"] = "http://%s:5984" % (config.Agent.hostName)
 config.WorkQueueManager.queueParams["WorkPerCycle"] = 100  # don't pull more than this number of elements per cycle
+config.WorkQueueManager.queueParams["drain_mode"] = False
```

Comment:
It has to be named `DrainMode`, for the agent config.
 `drain_mode` is the key name in wmstats.